### DISCUSSION TITLE
SOH estimation

### DISCUSSION THREAD

**Discussion by konkeytonton** [pybamm-team/PyBaMM#3553] [Q&A]
I want to estimate the SOH of a battery through the life cycle of a nano satellite.I have the experiment cycle ready as well as the parameters.Its just that I'm unable to find a a way of calculating SOH.I thought about using internal resistance via Local ECM resistance, however I can't do that without the End of Life internal resistance being available.I also considered using the total lithium lost and converting it to Ah.However that does not give us the overall capacity lost
Is there any other method to do it or am I missing something obvious here

**Comment by TomTranter** [comment]
Assuming you have some data already you have two options to fit degradation models and both require a good model (i.e. accurate parameters) for a fresh cell to start with:

1) Try to reproduce the capacity loss and IR increase by manually fitting a series (every 10-50 cycles depending on rate of change) of models to aged data and adjust the lithium inventory and active material and sei layer thickness, porosity etc. manually to see if patterns emerge that fit your data. You are fitting the degradation effects rather than the causes which result in changes to the above parameters.
2) Turn on the degradation models, explore different combinations and parameter values to dial up and down the causes according to physical phenomena and run the full set of simulations (every cycle) to reproduce your data. This will take much longer and involves more trial and error to tune degradation parameters. However, this may give you more insight as it will become more apparent which phenomena is causing the effect that is most prominent in your data.

It's probably sensible to start with approach 1 and then move to 2 if you need more insight.

This really does all depend on what information you are trying to gather and what the purpose of gaining this insight is going to be. For example if you have a temperature dependent degradation mechanism that is of most concern you would be advised to identify that relationship which may become more apparent or easy to extrapolate when you look into modelling the causes of the degradation rather than just capturing the effects under one set of operating conditions. In other words approach 2 is more generalisable.

**Reply by konkeytonton** [reply]
I don't have any degradation data myself.I wanted to simulate for an upcoming mission.if I can find degradation data based on the parameters i.e the LG cell it's based 
Would it be possible for me to extrapolate it to different conditions?

**Reply by TomTranter** [reply]
Physical models should extrapolate better than any other if you have a good enough range in your data to start with. If interested in some help with any of this you can contact info@ion-works.com for consultancy and advice
